lib/libc/minimal/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ zephyr_library_sources(
   source/stdlib/malloc.c
   source/stdlib/bsearch.c
   source/stdlib/exit.c
+  source/stdlib/rand.c
```

Comment:
Need to remove this as per the `zephyr_library_sources_ifdef` below.

Reply:
done